themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +35,30 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group'] && strtolower($edsConfig->AuthorResultListInfo->ResultListPage ?? 'Long') == 'short'): ?>
+                    <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+		    <?php $authorCount = count($authors) ?>
+		    <?php $authorNameLimit = $edsConfig->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                    <?php if (!empty($authors)): ?>
+                        <b><?=$this->transEsc($item['Label'])?>:</b>
+                        <?php foreach ($authors as $i => $author): ?>
+                            <a href="<?=$this->record($driver)->getLink('author', $this->highlight($author, null, true, false))?>"><?=$this->highlight($author)?></a><?php
+                            if ($authorCount == ($i + 1)) {
+                              break;
+                            } elseif (($i + 1) >= $authorNameLimit) {
+                              echo ' ' . $this->transEsc('more_authors_abbrev');
+                              break;
+                            } else {
+                              echo '; ';
+                            }
```

Comment:
You may prefer to extract the ';' logic into CSS (considering it a stylistic element, and perhaps something that would be localized differently), i.e. wrap each `<a>` with some `<span class="author">` and then something like

`.search-results-eds .result .author:not(:last-of-type):after {
    content: '; ';
}`

Then I think the conditional (after the loop) would only have to to deal with adding the more_authors_abbrev if $authorCount > $authorCountLimit.  That said, there's nothing wrong with the way it's implemented now.

Reply:
that's great to hear that you like this feature @maccabeelevine !

I've added this enhancement in my latest follow-up. I am not sure how the CSS works for applying the author name delimiter on the combined result page. So I'll be keen to hear how it looks for you.

Reply:
Yup it works in a combined column!

![image](https://github.com/user-attachments/assets/b40603d9-a56b-4fcf-aea6-c441955d28c3)

Reply:
fabulous, thanks for testing @maccabeelevine !